components/validator/guardian.go

```diff
@@ -164,59 +113,65 @@ func (g *Guardian) handleValidationRequested(ctx context.Context) {
 	}
 }
 
-// TODO(pangssu): Retry a failed or missed transaction.
+func (g *Guardian) tryConfirmTransaction(ctx context.Context, event *bindings.SecurityCouncilValidationRequested) error {
```

Comment:
I suggest that just return error in `tryConfirmTransaction` (do not log here) and log error in caller side with error message.